ports/osqp-eigen/vcpkg.json

```
@@ -0,0 +1,25 @@
+{
+  "name": "osqp-eigen",
+  "version": "0.7.0",
+  "port-version": 2,
+  "description": "Simple Eigen-C++ wrapper for OSQP library",
+  "homepage": "https://robotology.github.io/osqp-eigen/",
+  "dependencies": [
+    {
+      "name": "eigen3",
+      "version>=": "3.4.0"
+    },
```

Normally, port manifests don't specify minimum versions in all details.
```suggestion
    "eigen3",
```